WickedEngine/wiECS.h

```diff
@@ -346,6 +404,48 @@ namespace wi::ecs
 		// Disallow this to be copied by mistake
 		ComponentManager(const ComponentManager&) = delete;
 	};
+
+	// This is the class to store all ComponentManagers,
+	// this is useful for bulk operation of all attached components within an entity
+	class ComponentLibrary{
+	public:
+		uint64_t libraryVersion;
+		wi::vector<ComponentManager_Interface*> componentManagers;
```

Comment:
Should be `wi::vector<std::unique_ptr<ComponentManager_Interface>> componentManagers;` instead, to avoid memory leak on destroy.